templates/odoo.conf.j2

```diff
@@ -8,3 +8,7 @@ addons_path = {{ odoo_role_odoo_path }}/addons,{{ odoo_role_odoo_modules_path }}
 
 ; Master password to manage dbs
 admin_passwd = {{ odoo_role_odoo_db_admin_password }}
+
+; HTTP server settings
+http_interface = {{ odoo_role_odoo_http_interface }}
+proxy_mode = {{ odoo_role_odoo_proxy_mode }}
```

Comment:
This config var surprises me. I didn't know it was necessary.
https://www.odoo.com/documentation/11.0/reference/cmdline.html